sentinel-extension/sentinel-metric-exporter/src/main/java/com/alibaba/csp/sentinel/metric/exporter/jmx/MetricBean.java

```diff
@@ -0,0 +1,136 @@
+/*
+ *  Copyright 1999-2021 Alibaba Group Holding Ltd.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+
+package com.alibaba.csp.sentinel.metric.exporter.jmx;
+
+import com.alibaba.csp.sentinel.node.metric.MetricNode;
+
+/**
+ * the MetricBean for JMX expose.
+ *
+ * @author chenglu
+ * @date 2021-07-01 20:02
+ * @since 1.8.3
+ */
+public class MetricBean implements MetricMXBean {
+    
+    private String resource;
+    
+    /**
+     * Resource classification (e.g. SQL or RPC)
+     */
+    private int classification;
+    
+    private long timestamp;
+    
+    private long passQps;
+    
+    private long blockQps;
+    
+    private long successQps;
+    
+    private long exceptionQps;
+    
+    private long rt;
+    
+    private long occupiedPassQps;
+
+    private int concurrency;
+    
+    @Override
+    public String getResource() {
+        return resource;
+    }
+    
+    @Override
+    public int getClassification() {
+        return classification;
+    }
+    
+    @Override
+    public long getTimestamp() {
+        return timestamp;
+    }
+    
+    @Override
+    public long getPassQps() {
+        return passQps;
+    }
+    
+    @Override
+    public long getBlockQps() {
+        return blockQps;
+    }
+    
+    @Override
+    public long getSuccessQps() {
+        return successQps;
+    }
+    
+    @Override
+    public long getExceptionQps() {
+        return exceptionQps;
+    }
+    
+    @Override
+    public long getRt() {
+        return rt;
+    }
+    
+    @Override
+    public long getOccupiedPassQps() {
+        return occupiedPassQps;
+    }
+    
+    @Override
+    public int getConcurrency() {
+        return concurrency;
+    }
+    
+    /**
+     * reset the MBean value to the initialized value.
+     */
+    public void reset() {
+        this.blockQps = 0;
+        this.passQps = 0;
+        this.timestamp = System.currentTimeMillis();
+        this.exceptionQps = 0;
+        this.occupiedPassQps = 0;
+        this.successQps = 0;
```

Comment:
The reset() is not completely paired with set(). No need to reset other properties like timestamp, concurrency, rt, and so on?

Reply:
I will keep it same with set.